### ISSUE TITLE
Android build system setup and manifest

### ISSUE THREAD

**Issue by rpavlik** [KhronosGroup/OpenXR-Tutorials#37]
I left a number of comments on https://github.com/KhronosGroup/OpenXR-Tutorials/commit/7738eb62a1b65b5e2393b3ea776aad0641dac2e2

My overall concern is that there is a lot of prose being spent on things to remove and add, when:

- Google frequently changes the Android gradle plugin so it's hard to keep up with the contents of the template (stuff to remove)
- Several of the changes actually add back deprecated Android gradle plugin usage
- Only a few details are actually important here for the details of using OpenXR in native code on Android: the other parts are just "normal native code on android" things or "stuff you don't actually want to do but that's what we did to match our example"

Basically in nearly every release we have to adjust stuff in our Gradle files to work with the latest Android Studio changes. It's preferable, at least to me, if the text sticks to just those things that are OpenXR-specific (which *haven't* had to change in quite a while) so updates can just be to the build, not to the text as well.

cc @rbessems

**Comment by rpavlik** [comment]
See also: I just made updates to the androidteapots sample so that it closely matches both the upstream NDK samples and best practices for Android Gradle  Plugin 7.4.2.  https://gitlab.freedesktop.org/monado/demos/androidteapots/-/tree/updates

**Comment by rpavlik** [comment]
minSdkVersion should be 24, not 29, or you'll exclude a bunch of devices.

**Comment by AndrewRichards-Code** [comment]
I have committed 9ae39fa and c3b73f2 to reduce the text complexity of the Android build files (showing them as diffs) and amending issues in the build files themselves.

**Comment by rvkennedy** [comment]
Closing for now, re-open if still pertinent.
